tests/actionsFactory.spec.ts

```diff
@@ -58,6 +58,51 @@ describe("actionsFactory test", () => {
         }); 
     });
 
+    describe("create Snippet action", () => {
+        it("suceeds with all the mandatory params", () => {
+            const actionJson = {
+                [ActionJsonKey.ActionType]: ActionType.Snippet,
+                id: "id",
+                snippetName: "name",
+                contributorId: "contributorId",
+                context: {},
+                isNonInteractive: true
+            };
+            const action = ActionsFactory.createAction(actionJson);
+            expect(action instanceof SnippetAction).to.be.true;
+            expect((action as SnippetAction).snippetName).to.be.equal("name");
+            expect((action as SnippetAction).contributorId).to.be.deep.equal("contributorId");
```

Comment:
`to.be.deep.equal` is not proper english
I think it should be `to.deep.equal`

note it does not change the runtime, it is style/grammar only